kin-base/src/main/java/kin/base/KeyPair.java

```diff
@@ -31,7 +31,7 @@
  */
 public class KeyPair {
 
-  private static final EdDSANamedCurveSpec ed25519 = EdDSANamedCurveTable.getByName("ed25519-sha-512");
+  private static final EdDSANamedCurveSpec ed25519 = EdDSANamedCurveTable.getByName("ed25519");
```

Comment:
use the library const `EdDSANamedCurveTable.ED_25519_CURVE_SPEC` see https://github.com/stellar/java-stellar-sdk/commit/3826aa224bad802c4524858c75f4c2e65ba5ee9f